hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -425,8 +425,11 @@ class ip_env_cfg extends cip_base_env_cfg #(.RAL_T(ip_reg_block));
 ```
 
 ### Security Verification for REGWEN CSRs
-If the REGWEN CSR is HW read-only, it can be fully verified by the common [csr_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/csr_tests.hjson).
-If it’s a HW updated CSR, users need to write a test to verify it separately since cip_lib and dv_base_reg can’t predict its value.
+If the REGWEN CSR meets the following criteria, it can be fully verified by the common [csr_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/csr_tests.hjson).
+ - The REGWEN CSR and its related lockable CSRs are HW read-only registers.
+ - The related lockable CSRs are not WO type, otherwise, read value is always 0 and CSR tests can't really verify if the write value is taken or not.
```

Comment:
Nit:
```suggestion
 - The related lockable CSRs are not WO type, otherwise the value read is always 0 and CSR tests can't really verify if the write value is taken or not.
```
(doing it the other way around makes it ambiguous whether it's the "read value" (pronounced REED, and meaning "the value that came from a read") or the "read value" (pronounced RED, meaning "the value that was read"). It doesn't really matter which, but the sentence reads better if there's only one possible interpretation.

Reply:
Done.